services/core-api/app/api/email_preview/resources/email_preview_resource.py

```diff
@@ -0,0 +1,230 @@
+"""
+Email Preview Resource
+Provides development endpoints for previewing email templates with test data
+"""
+
+from flask import current_app, request, jsonify, Response
+from flask_restx import Resource, reqparse
+
+from app.extensions import api
+from app.api.utils.access_decorators import requires_any_of
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.access_decorators import VIEW_ALL, MINESPACE_PROPONENT
+from app.api.constants import CORE_PURPLE_LOGO_BASE64_ENCODED, MINESPACE_LOGO_BASE64_ENCODED, BC_GOV_LOGO_BASE64_ENCODED
+
+
+class EmailPreviewResource(Resource, UserMixin):
+    """
+    Email Preview API endpoint for development
+    """
+    
+    @requires_any_of([VIEW_ALL, MINESPACE_PROPONENT])
+    def get(self, template_name):
+        """
+        Preview an email template with test data
+        GET /api/email-preview/<template_name>
+        """
+        try:
+            # Get template with proper path
+            template_path = f"email/{template_name}"
+            template = current_app.jinja_env.get_template(template_path)
```

Comment:
I'm wondering if this could be vulnerable for a path traversal attack (depending on how `get_template` implements it) - `f"email/.../.../.../etc/passwd`

Maybe we can define all the templates (and mapping to what file) as constants and use that as validation, think that would clean up the sending functionality a bit as well:

`EmailService.send_template_email(subject, recipients, EMAILS.INCIDENTS_RECEIVED_MS, email_context, cc=None)`